src/main/webapp/app/entities/alteration/alteration-update.tsx

```diff
@@ -60,7 +61,7 @@ export const AlterationUpdate = (props: IAlterationUpdateProps) => {
       props.alterationEntity.genes && props.alterationEntity.genes.length > 0
         ? props.alterationEntity.genes.map(gene => ({
             label: gene.hugoSymbol,
-            value: gene.id,
+            value: gene,
```

Comment:
I would stick with the id for value. If for some reason the reference changes we could end up in a weird state. If you do need to attach the actual gene, could you do it with another property?